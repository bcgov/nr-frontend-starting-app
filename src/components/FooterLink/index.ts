import styled from 'styled-components';

const FooterLink = styled.a`
  font-size: 0.813em;
  font-weight: normal;
  color: #fff;
  border-right: 1px solid #4b5e7e;
  padding-left: 5px;
  padding-right: 5px;
  text-decoration: none;

  &:hover
  {
    color: #fff;
    text-decoration: underline;
  }

  &:focus
  {
    outline: 4px solid #3B99FC;
    outline-offset: 1px;
  }
`;

export default FooterLink;
