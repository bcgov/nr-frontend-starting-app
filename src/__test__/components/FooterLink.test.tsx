/* eslint-disable no-undef */
import React from 'react';
import renderer from 'react-test-renderer';
import FooterLink from '../../components/FooterLink';

it('renders correctly', () => {
  const tree = renderer
    .create(<FooterLink />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
