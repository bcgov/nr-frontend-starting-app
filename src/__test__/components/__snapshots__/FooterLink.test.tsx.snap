// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the FooterLink component should match the snapshot 1`] = `
<a
  className="sc-bczRLJ keriIN"
/>
`;
